storage/tsdb/tsdb.go

```diff
@@ -117,6 +119,9 @@ type Options struct {
 	// The maximum timestamp range of compacted blocks.
 	MaxBlockDuration model.Duration
 
+	// The maximum size of each WAL segment file.
+	WALSegmentSize units.Base2Bytes
```

Comment:
I'd rather use `int` here.

Reply:
this is what i used before allowing the flag value to be a byte unit value thanks to https://github.com/alecthomas/units package. Then, i need WALSegmentSize to be the type units.Base2Bytes to make the parser works as expected

Reply:
this is the doc of https://godoc.org/github.com/alecthomas/kingpin#ArgClause.BytesVar i used here https://github.com/prometheus/prometheus/pull/5029/files#diff-fc1d26df0adf6edc1b9f25c878ab9ccaR169

Reply:
Ah right I missed that `Options` was used directly in the main func.